hw/bfin/bfin_devices.h

```diff
@@ -15,8 +15,10 @@
 #define mmr_idx(mmr)    (mmr_offset(mmr) / 4)
 #define mmr_name(off)   (mmr_names[(off) / 4] ? : "<INV>")
 
-#define HW_TRACE_WRITE() trace_bfin_reg_memory_write(addr, mmr_name (addr), size, value)
-#define HW_TRACE_READ()  trace_bfin_reg_memory_read(addr, mmr_name (addr), size)
+#define HW_TRACE_WRITE() trace_bfin_reg_memory_write(addr + s->iomem.addr, \
```

Comment:
thanks, i've squashed this change (and the state uncomments below) into the hardware patch